core/domain/platform_feature_services.py

```diff
@@ -114,15 +114,15 @@ def is_feature_enabled(feature_name):
 
 
 def update_feature_flag_rules(
-        feature_name, committer_id, commit_message, new_rule_dicts):
+        feature_name, committer_id, commit_message, new_rules):
     """Updates the feature flag's rules.
 
     Args:
         feature_name: str. The name of the feature to update.
         committer_id: str. ID of the committer.
         commit_message: str. The commit message.
-        new_rule_dicts: list(dict). A list of dict mappings of all fields
-            of PlatformParameterRule object.
+        new_rules: list(PlatformParameterRule). A list of dict of
```

Comment:
This looks like this might need to be updated a bit more; this is no longer a list of dictionaries.

Perhaps instead: ``A list of PlatformParameterRule objects to update.`` might be clearer?

Reply:
Updated.